src/commands/analytics/display-analytics.ts

```diff
@@ -98,7 +101,7 @@ export async function displayAnalytics({
 }
 
 function displayAnalyticsScreen(data: FormattedData): void {
-  const screen = new Widgets.Screen({})
+  const screen: Widgets.Screen = new ScreenWidget({})
```

Comment:
Would this be implicit by the return type of ScreenWidget? does it need to be explicitly added to `const screen: Widgets.Screen`?

Reply:
TS is ignoring the import so `ScreenWidget` is `any`. But it should be the same value as what I was expecting `Widgets.Screen` to be, except importing it fails because apparently `Widgets` ends up as `undefined`.

This way we get a typed `screen` variable, mitigating the impact of the ts-ignore.